mllib/src/main/scala/org/apache/spark/ml/classification/LinearSVC.scala

```diff
@@ -355,6 +368,19 @@ object LinearSVCModel extends MLReadable[LinearSVCModel] {
 }
 
 /**
+ * Abstraction for Linear SVC Training results.
+ * Currently, the training summary ignores the training weights except
+ * for the objective trace.
+ */
```

Comment:
I'm not sure why we didn't include the weight column in the summary. Maybe add an option[String] ?